app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    new_file = command.split()
+
+    if new_file == 3:
```

Comment:
what does it mean `new_file == 3`? may it be?
```suggestion
    if len(new_file) == 3:
```